### ISSUE TITLE
Bugs on wsj.com

### ISSUE THREAD

**Issue by AFlyingBird90** [magnolia1234/bypass-paywalls-chrome-clean#53]
On wsj.com, clicking the extension shows "reload page to use this extension" even after repeated reloading.

**Comment by magnolia1234** [comment]
What do you mean, clicking the extension?
Can you upload a screenshot of the issue?
For me WSJ is working just fine.

**Comment by AFlyingBird90** [comment]
![Screen Shot 2020-03-16 at 8 34 42 PM](https://user-images.githubusercontent.com/22230037/76758929-a56fb080-67c5-11ea-8da7-21b4a344c6e4.png)
The paywall-bypassing works for me. The problem is on wsj.com when I click the extension, it shows "reload page to use the extension" instead of "bypass paywall clean v1.6.4.4". And the problem only appear on wsj.com, not on economist.com.

**Comment by magnolia1234** [comment]
I have never seen this before.
Maybe another proxy-issue?

**Comment by marathone** [comment]
On Mon, Mar 16, 2020 at 06:32:30AM -0700, magnolia1234 wrote:
> I have never seen this before.
> Maybe another proxy-issue?
>

There is an option in Google-Chrome to not have extensions automatically 
interact with a page unless clicked on - This is where I see the 'reload 
page' for the extension to work.

The only reason I use this for some extensions, is to remove the possibility 
they may interact negatively with web login components.

Maybe the OP could confirm if this is the case - that he has his extension 
set up that way?

**Comment by magnolia1234** [comment]
Indeed that setting shows this message also, but here the extension is already active on the tab/site (ON-badge is visible).

**Comment by AFlyingBird90** [comment]
![Screen Shot 2020-03-17 at 8 12 33 AM](https://user-images.githubusercontent.com/22230037/76810398-e351f080-6828-11ea-80c4-23b888d3500e.png)
![Screen Shot 2020-03-17 at 8 23 52 AM](https://user-images.githubusercontent.com/22230037/76810400-e64ce100-6828-11ea-910d-913755ff5c3b.png)
![Screen Shot 2020-03-17 at 8 23 35 AM](https://user-images.githubusercontent.com/22230037/76810405-ea78fe80-6828-11ea-876b-960daedd1bea.png)
I have disabled all other extensions. Mouse over the extension icon on wsj.com shows "Bypass paywalls clean wants access to this site" while I have already granted access (and the paywall-bypassing functions well), thus after clicking it requires reloading. On economist.com, mouse over shows "... has access to this site".
